detekt/wrapper/src/main/kotlin/io/buildfoundation/bazel/rulesdetekt/wrapper/Executable.kt

```diff
@@ -0,0 +1,37 @@
+package io.buildfoundation.bazel.rulesdetekt.wrapper
+
+import java.io.File
+import java.io.FileOutputStream
+import java.io.PrintStream
+import java.nio.charset.Charset
+
+internal interface Executable {
+
+    fun execute(args: Array<String>): Result
+
+    class DetektImpl(private val detekt: Detekt) : Executable {
+
+        override fun execute(args: Array<String>): Result {
+            val outputPrinter = PrintStream(Streams.DevNullOutputStream().buffered())
+
+            val errorPrinterFile = File.createTempFile("detekt-errors", null)
```

Comment:
Please specify the directory here, otherwise this will write to `$TMPDIR` (which I assume will be real user `TMPDIR` unless Bazel exports own value) if running without sandboxing, not perfect

Reply:
Good call! In fact, `File.createTempFile` is a bit better from the parallel execution standpoint (temporary files are unique). Regular `File` one, however, risks to be shared between threads. I’ve replaced it with the `ByteArrayOutputStream` since we are forced to dump everything into memory anyway.